index.js

```diff
@@ -62,6 +62,28 @@ server.post('/api/users', (req,res) => {
 
```

Comment:
could you be more descriptive as to why its not working perfectly, looks ok to me